javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,41 @@
+const jWidth = 5;
+const jDeep = 11;
+const jHeight = 8;
+const jVolumeInMeters = jWidth * jHeight * jDeep;
+const jGardenSizeInM2 = 70;
+const jHouseCost = 1000000;
+jHousePrice = jVolumeInMeters * 2.5 * 1000 + jGardenSizeInM2 * 300
+//console.log(' Julia house price ' + jHousePrice)
+let jHousey;
+if(jHouseCost > jHousePrice){
+    jHousey = 'Julia is Paying more';
+}
+else if(jHouseCost=jHousePrice){
```

Comment:
`=` is called the assignment operator, which is used to assign values to variables. I think what you meant to use here is `==`, the equality operator.

Reply:
ya i got it...it was a mistake but unfortunately i have merged so what can i do now :(

Reply:
It's ok, as long as you got what was wrong and learned how to fix it :)

You can create a PR from your repo, fix the issue, then merge it. Let me know if you need help doing that.

Reply:
Thanks for reviewing my hw. I have made changes and remerged . :)

Reply:
You're welcome :)